deploy/examples/create-external-cluster-resources.py

```diff
@@ -826,7 +955,27 @@ def validate_rados_namespace(self):
             raise ExecutionFailureException(
                 ("The provided rados Namespace, '{}', is not found in the pool '{}'").format(
                     rados_namespace, rbd_pool_name))
-
+    
+    def get_rgw_fsid(self):
+        access_key = self.out_map['RGW_ADMIN_OPS_USER_ACCESS_KEY']
+        secret_key = self.out_map['RGW_ADMIN_OPS_USER_SECRET_KEY']
+        rgw_endpoint = self._arg_parser.rgw_endpoint
+        request_url = "http://"
+        if self._arg_parser.rgw_tls_cert_path:
+            request_url = "https://"
+        request_url = request_url + rgw_endpoint + "/admin/info" + "?" + "format=json"    
```

Comment:
I am assuming this curl request sending to rgw_endpoint, so if it is `HTTPS` then either need to attach certificate or use `-k` for insecure flag for curl command. Otherwise request will fail

Reply:
I am not sure about networking, 
@leseb can you assist with what else to do for HTTPS request/

Reply:
It depends, where is HTTP client we build to run the request?

Reply:
OKay, the URI is made like this only  `  request_url = request_url + rgw_endpoint + "/admin/info" + "?" + "format=json"   `